src/main.ml

```diff
@@ -223,10 +223,7 @@ let cmd =
   let port =
     let doc = "Local port to listen for connections on" in
     Arg.(value & opt int Consts.standard_nbd_port & info [ "port" ] ~doc) in
-  let xen_api_uri =
-    let doc = "The URI to use when making XenAPI calls. It must point to the pool master, or to xapi's local Unix domain socket, which is the default." in
-    Arg.(value & opt string Consts.xapi_unix_domain_socket_uri & info [ "xen-api-uri" ] ~doc) in
```

Comment:
@mseri I'm happy to reintroduce this command line argument if you reckon we'll need it, but it will have to point to the local Unix domain socket, because we login again in the signal handler.